docs/instances.mdx

```diff
@@ -1,7 +0,0 @@
----
```

Comment:
This is still relevant for vanilla emotion so probably should be kept here (but reworded)

Reply:
I think what I'm going to do is add some documentation about instances in the `@emotion/css` and SSR docs instead of this page.